pkg/targetallocator/container.go

```diff
@@ -50,6 +52,9 @@ func Container(cfg config.Config, logger logr.Logger, otelcol v1alpha1.OpenTelem
 	if otelcol.Spec.TargetAllocator.PrometheusCR.Enabled {
 		args = append(args, "--enable-prometheus-cr-watcher")
 	}
+	if len(otelcol.Spec.TargetAllocator.AllocationStrategy) > 0 {
+		args = append(args, fmt.Sprintf("--allocation-strategy=%s", otelcol.Spec.TargetAllocator.AllocationStrategy))
```

Comment:
Maybe we should put this in the config file we have already 🤔.

Reply:
Which file would that be?

Reply:
Config of the TA: 
https://github.com/open-telemetry/opentelemetry-operator/blob/main/cmd/otel-allocator/config/config.go#L33

Operator code which generates this file: https://github.com/open-telemetry/opentelemetry-operator/blob/19c186df1b89a3898169a16f91bd13fab5a6931f/pkg/collector/reconcile/configmap.go#L94